.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Reference: https://help.github.com/en/articles/about-code-owners
+# Order is important: later rules override preceding rules
+* @mesosphere/kommander
```

Comment:
Is this the correct syntax?

Reply:
It is, but the team probably isn't public, or perhaps doesn't have write access to this repo. @fatz Can you help out with this please to ensure it has the access it needs?